scripts/bundles/plugins/typescript-source-plugin.ts

```diff
@@ -140,7 +140,6 @@ async function bundleTypeScriptSource(tsPath: string, opts: BuildOptions): Promi
   const o: string[] = [];
   o.push(`// TypeScript ${opts.typescriptVersion}`);
   o.push(`import { IS_NODE_ENV } from '@environment';`);
-  o.push(`process.browser = !IS_NODE_ENV;`);
```

Comment:
Does `process.browser` get relied on somehow? E.g. in the TS source code? I'm curious what this was doing beforehand.

Also, do we need the import on `IS_NODE_ENV` still?

Reply:
Oh yeah we could remove that here too (the `IS_NODE_ENV` import)

I'm going to do a separate PR to remove the `IS_NODE_ENV` and `IS_BROWSER_ENV` variables entirely, it's a little more involved for that though

Reply:
and I don't believe that we have any dependency on `process.browser`

Reply:
FWIW, TS has the following in its source:
```
  function isNodeLikeSystem() {
    return typeof process !== "undefined" && process.nextTick && !process.browser && typeof module === "object";
  }
```
It _may_ make sense to just set this value to `false` explicitly, although IDK if there's another "thing" that could be setting it

Reply:
I don't think we need to do that, most people using typescript in a node environment don't end up needing to do anything in particular to tell typescript 'this is node, do node things' so I don't think we have a particular need to either - this stuff was I think here purely for indicating the opposite case (in appropriate conditions)

Reply:
Fair, that all makes sense to me!